pkg/server/profiles.go

```diff
@@ -112,7 +99,7 @@ func (s *ProfilesServer) GetProfiles(ctx context.Context, msg *pb.GetProfilesReq
 		return nil, fmt.Errorf("failed to get HelmRepository %q/%q: %w", s.HelmRepoNamespace, s.HelmRepoName, err)
 	}
 
-	ps, err := s.HelmChartManager.GetCharts(ctx, helmRepo, helm.Profiles)
+	ps, err := s.HelmCache.ListProfiles(ctx, helmRepo.Namespace, helmRepo.Name)
 	if err != nil {
```

Comment:
do we wanna delete the get request above? line 80

Reply:
I think that should be part of the next PR where we eliminate the use of a single helm repo, not?

Just because it would change how we return things. It would remove at least one of the errors. Either the failed to scan or the not found being returned as a 200. Which is fine by me to get it nuked. But I don't know what the UI expect.

Reply:
Same for `GetProfileValues`.

Reply:
sure